src/installscreen.cpp

```diff
@@ -56,7 +56,7 @@ InstallScreen::InstallScreen(PackageBackend *backend, QString *tmpFileName, QObj
     installLayout->setSpacing( 20 );
 
     foreach( QUrl iter, m_backend->repositories() ) {
-        QLabel *sourceLabel = new QLabel( QString( "Added Source: %1" ).arg( iter.toString() ) );
+        QLabel *sourceLabel = new QLabel( i18n("Added Source: %1", iter.toString()) );
```

Comment:
Yes, as I said, it's not wrong, but it's not common to use it like that afaik. Would you mind changing that to the usual form?